src/material-examples/stepper-overview/stepper-overview-example.html

```diff
@@ -1,5 +1,6 @@
-<button mat-raised-button (click)="isLinear = true" id="toggle-linear">Enable linear mode</button>
-
+<button mat-raised-button (click)="isLinear = !isLinear" id="toggle-linear">
+  {{!isLinear?'Enable linear mode':'Disable linear mode' }}
+</button>
 <mat-horizontal-stepper [linear]="isLinear" #stepper="matHorizontalStepper">
```

Comment:
Again don't know why it was like this, but I think just `#stepper` should work, it grabs the component by default